### DISCUSSION TITLE
Raspberry Pi Zero 2 W

### DISCUSSION THREAD

**Discussion by andytt11** [iiab/iiab#3846] [Q&A]
I've got IIAB medical running on a Zero W and have tried to flash 8.1 Small onto another drive to use my new Zero 2 W to no avail. I cant get the medical version to run either but I assume the 64 bit is the reason. Is there something wrong with using rpi-imager --repo http://iiab.io/fast.json for the 2 W. WIFI connection never shows up to diagnose.

**Comment by holta** [comment]
> flash 8.1

Apologies that image is about a year old!

A pre-release of IIAB 8.2 would be much better if you can possibly install that instead — noting that IIAB 8.2 pre-releases are very solid, with its final release becoming official very shortly in a few weeks.

If so @andytt11, would you (1) have the ability to install 64-bit Raspberry Pi OS Lite onto a blank microSD card using Raspberry Pi Imager?

And if so (a keyboard and screen might be helpful!), would you then (2) be able to try installing IIAB directly, by logging into that Raspberry Pi OS Lite, and then running... ?

```
curl iiab.io/install.txt | bash
```

**Reply by andytt11** [reply]
No apologies needed. I'll install OS lite and give it a go.

**Reply by holta** [reply]
Thanks @andytt11 !  When it asks you "PLEASE PICK (1) 🚵 SMALL (2) 🚢🚣 MEDIUM or (3) 🚂🚃🚃 LARGE (NEEDS 2+ GB RAM).  Or try (m) for 🚑 MEDICAL." you might possibly type "m" for Medical?

If so, your [/etc/iiab/local_vars.yml](https://wiki.iiab.io/go/FAQ#What_is_local_vars.yml_and_how_do_I_customize_it?) will appear as follows:

https://github.com/iiab/iiab/blob/master/vars/local_vars_medical.yml

(At that point, feel free to customize your local_vars.yml file before proceeding!)

**Reply by andytt11** [reply]
raspios-bookworm-arm64-lite.img.xz image installed on sd card and setup in a Pi 3. when put into a zero 2 w just a constant boot flash. Not sure if i did something incorrect or not but I will try it one more time.

**Reply by holta** [reply]
Maybe verify that your Zero 2 W hardware is definitely working, using any other method?  (If possible!)

**Reply by andytt11** [reply]
Using official raspberry pi lite os from Pi imager worked. Not exactly sure why using the bookwork image did not but I appreciate the help.

**Reply by holta** [reply]
> Using official raspberry pi lite os from Pi imager worked.

64-bit "Lite" OS for Zero 2 W, right?

✅

**Reply by andytt11** [reply]
Correct.

**Comment by tim-moody** [comment]
I have the image used for the W2 offering on the wikipedia store if @holta wants to put it on iiab.io.

**Reply by holta** [reply]
Done: https://iiab.io/8.2/iiab-8.2-240923-raspios-64-starter-w2-g4dd927c.img (7.7 GiB / 7.8 GB)

**Comment by kholbekj** [comment]
@tim-moody does a 8.3 version exist of this image?

**Comment by tim-moody** [comment]
My most recent is in https://iiab.me/local_content/images/ from Dec 12.
